spec/fabricators/user_second_factor_fabricator.rb

```diff
@@ -7,6 +7,13 @@
   method UserSecondFactor.methods[:totp]
 end
 
+Fabricator(:user_second_factor_webauthn, from: :user_second_factor) do
+  user
+  data 'TODO (martin) - fill this in with a good approximation'
```

Comment:
No time like the present :)

Reply:
Argh thought I got rid of this one! Removing, as this fabricator is no longer relevant